tests/test_rtdc_feat_anc_plugin.py

```diff
@@ -964,6 +1044,8 @@ def test_pf_store_scalar_plugin_data_with_shape_issue_171():
         events = h5["events"]
         assert "circ_per_area" in events.keys()
         feat = h5["events"]['circ_per_area']
+        fshape = feat.shape
+        assert len(fshape) == 1
```

Comment:
I think a nicer way to look at the number of dimensions might be `assert feat.ndim == 1`.